app/views/users/show.html.erb

```diff
@@ -1,41 +1,71 @@
+                <%= link_to 'Edit Profile', edit_user_registration_path(@user), class: 'btn btn-default' %>
+              </div>
+              <div class="btn-group pull-right">
+                <%= link_to 'Dashboard', '#', class: 'btn btn-brand' %>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
   <div class="container">
     <div class="row">
-      <div class="col-xs-12 col-sm-4">
-        <h4>Account Info</h4>
-        <table class="table table-condensed">
-          <tbody>
-            <tr>
-              <th>Email:</th>
-              <td><%= @user.email %></td>
-            </tr>
-            <tr>
-              <th>Role:</th>
-              <td><%= @user.role %></td>
-            </tr>
-          </tbody>
-        </table>
-        <h4>What can this user see?</h4>
-        <table class="table table-condensed">
-          <tbody>
-            <tr>
-              <th>Free Content:</th>
-              <td><% if @user.is_free? %>Yes<% else %>No<% end%></td>
-            </tr>
-            <tr>
-              <th>Paid Content:</th>
-              <td><% if @user.is_paid? %>Yes<% else %>No<% end%></td>
-            </tr>
-            <tr>
-              <th>Admin Content:</th>
-              <td><% if @user.is_admin? %>Yes<% else %>No<% end%></td>
-            </tr>
-            <tr>
-              <th>Superadmin Content:</th>
-              <td><% if @user.is_superadmin? %>Yes<% else %>No<% end%></td>
-            </tr>
-          </tbody>
-        </table>
+      <div class="col-lg-7">
+
+        <div class="panel panel-default panel-table">
+          <div class="panel-heading">Account Info</div>
+          <table class="table table-deck">
+            <tbody>
+              <tr>
+                <th>Email:</th>
+                <td><%= @user.email %></td>
+              </tr>
+              <tr>
+                <th>Role:</th>
+                <td><%= @user.role %></td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
+        <div class="panel panel-default panel-table">
+          <div class="panel-heading">What can this user see?</div>
+          <table class="table table-deck">
+            <tbody>
+              <tr>
+                <th>Free Content:</th>
+                <td><% if @user.is_free? %>Yes<% else %>No<% end%></td>
+              </tr>
+              <tr>
+                <th>Paid Content:</th>
+                <td><% if @user.is_paid? %>Yes<% else %>No<% end%></td>
+              </tr>
+              <tr>
+                <th>Admin Content:</th>
+                <td><% if @user.is_admin? %>Yes<% else %>No<% end%></td>
+              </tr>
+              <tr>
+                <th>Superadmin Content:</th>
+                <td><% if @user.is_superadmin? %>Yes<% else %>No<% end%></td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
       </div>
     </div>
   </div>
```

Comment:
I think it makes more sense to use the `show` view **as** the `dashboard` itself. So in this PR there is a show view that gives some into about the account, which was really just written as a reference for us while we develop, and then there is a dashboard view that is linked to from the show view. When a user logs in, generally the reason they are there is to goto what is currently called the dashboard. If we get rid of the content in the current show view, and collapse the current dashboard view into it, then `eruditionapp.com/users/1` will render the dashboard.

This would require moving code from `users/dashboard.html.erb` to `uers/show.html.erb` and also moving code from the dashboard method to the show method in `users_controller.rb`